lib/shoryuken/polling/strict_priority.rb

```diff
@@ -68,8 +68,9 @@ def queue_paused?(queue)
       end
 
       def pause(queue)
-        return unless delay > 0
-        @paused_until[queue] = Time.now + delay
+        delay_time = delay(queue)
```

Comment:
@lucas-ibotta delay should be per group, not per queue.

Reply:
Going from the String of the queue name to the group delay required an extra helper... I'd like your thoughts on that implementation. I am testing this locally right now though.